src/openms/source/ANALYSIS/OPENSWATH/OpenSwathWorkflow.cpp

```diff
@@ -781,12 +802,17 @@ namespace OpenMS
                   swath_maps[i].lower < coordinates[j].mz && swath_maps[i].upper > coordinates[j].mz)
                 )
             {
-              // write MS1 chromatograms to disk
-              chromConsumer->consumeChromatogram( ms1_chromatograms[j] );
+#ifdef _OPENMP
```

Comment:
no need to ifdef an OpenMP pragma. This is only required for non-preprocessor OpenMP functionality.
Removing the ifdef improves readability of code.